ush/config_defaults.sh

```diff
@@ -1215,6 +1255,10 @@ MAKE_ICS_TN="make_ics"
 MAKE_LBCS_TN="make_lbcs"
 RUN_FCST_TN="run_fcst"
 RUN_POST_TN="run_post"
+
+ANAL_GSI_INPUT_TN="anal_gsi_input"
+ANAL_GSI_RESTA_TN="anal_gsi_resta"
```

Comment:
Can restart be spelled out here for clarity?

Reply:
Done